src/modules/ROOT/pages/changelog/202303/02.json

```diff
@@ -134,31 +136,34 @@
     },
     {
       "what": [
-        "automation sessions list sorting parameter in {APP_NAME_ADMIN}. The parameter was excepting the incorrect type of value."
+        "automation sessions list sorting parameter in {APP_NAME_ADMIN}.",
+        "The parameter was excepting the incorrect type of value."
       ],
       "refs": [
         "https://github.com/cortezaproject/corteza/pull/1141"
       ]
     },
     {
       "what": [
-        "copy of page layout in {APP_NAME_COMPOSE}. A page with layout couldn't be copied because of duplicate page handle value."
+        "copy of page layout in {APP_NAME_COMPOSE}.",
+        "A page with layout couldn't be copied because of duplicate page handle value."
       ],
       "refs": [
         "https://github.com/cortezaproject/corteza/pull/1230"
       ]
     },
     {
       "what": [
-        "page block visibility after being removed from Tabs configuration. If a block was added to Tabs configuration and removed afterwards, the  block wouldn't be shown in the page builder anymore."
+        "page block visibility after being removed from Tabs configuration.",
+        "If a block was added to Tabs configuration and removed afterwards, the  block wouldn't be shown in the page builder anymore."
```

Comment:
Merge these two sentences together.

something along the lines of `page blocks disappearing from the page builder after being removed from tabs ...`